src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary_tab.cljs

```diff
@@ -29,6 +29,52 @@
     (fn [k] {:userId k :accessLevel (acl-map k)})
     (keys acl-map)))
 
+(defn- create-section [& children]
+  [:div {:style {:padding "1em 0 2em 0"}} children])
+
+(defn- reload-attributes [props state]
+  (endpoints/call-ajax-orch
+    {:endpoint (endpoints/get-workspace (:workspace-id props))
+     :on-done  (fn [{:keys [success? get-parsed-response status-text]}]
+                 (swap! state assoc :server-response
+                   (if success? {:workspace (get-parsed-response)}
+                                {:error-message status-text}))
+                 (let [ws ((get-in @state [:server-response :workspace]) "workspace")]
+                   (swap! state assoc :attrs-list (mapv (fn [[k v]] [k v]) (ws "attributes")))))})
+  )
+
+(defn- add-update-attributes [props attrs]
+  (let [workspace-id (:workspace-id props)
+        add-update-ops (for [pair attrs]
+                         {:op                 "AddUpdateAttribute"
```

Comment:
Don't line these up—it's painful to maintain this. Just one space after keyword.